test/statsmodel.jl

```diff
@@ -233,6 +235,13 @@ Base.show(io::IO, m::DummyModTwo) = println(io, m.msg)
     m2 = fit(DummyModTwo, f, d)
     # make sure show() still works when there is no coeftable method
     show(io, m2)
+
+    # one final termnames check
+    # note that `1` is still a ConstantTerm and not yet InterceptTerm
+    # because apply_schema hasn't been called
+    @test termnames(@formula(y ~ 1 + log(x) * y + (1+x|g)))[2] == 
+          ["1", "log(x)", "y", "log(x) & y", "(1 + x) | g"]
+          
 end
```

Comment:
Can you add tests that call `termnames` directly on individual terms?

Reply:
done